Catalog/Examples/Networking/Combine/CombineNetworkingPresenter.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CombineNetworkingPresenter.swift
+//  Catalog
+//
+//  Created by Zvonimir Medak on 29.10.2021..
+//  Copyright (c) 2021 Infinum. All rights reserved.
+//
+//  This file was generated by the 🐍 VIPER generator
+//
+
+import Foundation
+import Combine
+import Alamofire
+
+@available(iOS 13, *)
+final class CombineNetworkingPresenter {
+
+    // MARK: - Private properties -
+
+    private unowned let view: CombineNetworkingViewInterface
+    private let interactor: CombineNetworkingInteractorInterface
+    private let wireframe: CombineNetworkingWireframeInterface
+
+    // MARK: - Lifecycle -
+
+    init(
+        view: CombineNetworkingViewInterface,
+        interactor: CombineNetworkingInteractorInterface,
+        wireframe: CombineNetworkingWireframeInterface
+    ) {
+        self.view = view
+        self.interactor = interactor
+        self.wireframe = wireframe
+    }
+}
+
+// MARK: - Extensions -
+
+@available(iOS 13, *)
+extension CombineNetworkingPresenter: CombineNetworkingPresenterInterface {
+    func configure(with output: CombineNetworking.ViewOutput) -> CombineNetworking.ViewInput {
+        let userInfo = Publishers.CombineLatest(output.email, output.password)
+
+        let didLogin = output
+            .login
+            .combineLatest(userInfo)
+            .map { $0.1 }
+            .setFailureType(to: AFError.self)
+            .flatMap { [unowned self] email, password in
+                return interactor
+                    .login(email, password)
+                    .mapTo(true)
```

Comment:
Since we're working with bools that we print in the VC, it makes more sense to catch the error here and map it to false 🙂

Reply:
flatMap was having issues if I set the replaceError(with:)  inside the curly braces (available from iOS 14 error), so I added it after the flatMap